<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>GeeCee</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../../SVN/toolz/SimpleView/icons.qrc" >:/new/prefix1/Retro Mario World_32.png</iconset>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="plotTab" >
       <attribute name="title" >
        <string>Plot</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Spectrum Index:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="indexHSlider" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="indexSpinBox" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Spectrum Location:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="specNameEdit" >
            <property name="readOnly" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QSplitter" name="splitter" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="MPL_Widget" native="1" name="plotWidget" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>500</width>
               <height>350</height>
              </size>
             </property>
            </widget>
            <widget class="MPL_Widget" native="1" name="plotWidget2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize" >
              <size>
               <width>500</width>
               <height>350</height>
              </size>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QListWidget" name="listWidget" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="baseSize" >
               <size>
                <width>100</width>
                <height>250</height>
               </size>
              </property>
              <property name="layoutDirection" >
               <enum>Qt::RightToLeft</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame" >
              <property name="minimumSize" >
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QGridLayout" >
               <item row="0" column="0" >
                <layout class="QGridLayout" >
                 <item row="0" column="0" >
                  <layout class="QGridLayout" >
                   <property name="horizontalSpacing" >
                    <number>6</number>
                   </property>
                   <item row="0" column="0" >
                    <widget class="QLabel" name="label_3" >
                     <property name="maximumSize" >
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text" >
                      <string>SIC m/z</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1" >
                    <widget class="QSpinBox" name="sicMZSB" >
                     <property name="maximumSize" >
                      <size>
                       <width>75</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="minimum" >
                      <number>1</number>
                     </property>
                     <property name="maximum" >
                      <number>400</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="1" column="0" >
                  <widget class="QPushButton" name="pltSICBtn" >
                   <property name="maximumSize" >
                    <size>
                     <width>75</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text" >
                    <string>Plot SIC</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" >
                  <widget class="QPushButton" name="plotTICBtn" >
                   <property name="maximumSize" >
                    <size>
                     <width>75</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text" >
                    <string>Plot TIC</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="optionsTab" >
       <attribute name="title" >
        <string>Options</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget" >
        <property name="geometry" >
         <rect>
          <x>11</x>
          <y>13</y>
          <width>261</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QCheckBox" name="loadmzXMLCB" >
           <property name="text" >
            <string>Load mzXML</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QCheckBox" name="plotPkListCB" >
           <property name="text" >
            <string>Plot Peak List</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QCheckBox" name="plotLegendCB" >
           <property name="text" >
            <string>Plot Legend</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="loadDirBtn" >
           <property name="text" >
            <string>Load Directory</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QCheckBox" name="excludeLIFTCB" >
           <property name="text" >
            <string>Exclude LIFT MS/MS Files</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="pkListTab" >
       <attribute name="title" >
        <string>Peak List</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open" />
   </widget>
   <addaction name="menu_File" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPL_Widget</class>
   <extends>QWidget</extends>
   <header>mpl_pyqt4_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../SVN/toolz/SimpleView/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>indexHSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>553</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>601</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>indexSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>indexHSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>601</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>553</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
